src/plugins/table/table.yml

```diff
@@ -19,3 +19,18 @@ cellAction_insertColumnAfter:
 cellAction_deleteColumn:
   en: Delete column
   de: Spalte löschen
+cellAction_connectToRowBefore:
+  en: Connect with row before
+  de: Mit Zeile davor verbinden
+cellAction_connectToRowAfter:
+  en: Connect with row after
+  de: Mit Zeile danach verbinden
+cellAction_connectToColumnBefore:
+  en: Connect with column before
```

Comment:
How about "Connect with previous column", "Connect with following row" etc. for a more natural phrasing? 🤔

Reply:
Actually I realized the "row" and "columns" might be confusing for the user. So I renamed them to "Connect with cell above/below/on the left/on the right" and the counterpart in the German (I left the enumeration values as they were though, I think technically they're not wrong)

Reply:
That's true, "cells" are what the user connects, but I agree that programmatically "rows" and "columns" are what we process. Sounds good.